src/bscPlugin/completions/CompletionsProcessor.ts

```diff
@@ -24,6 +24,9 @@ import { createIdentifier } from '../../astUtils/creators';
 import type { FunctionExpression } from '../../parser/Expression';
 import { LogLevel } from '../../Logger';
 
+
+const SCOPES_FOR_COMPLETION = 5;
```

Comment:
Is there a reason you went with 5 instead of 3 like we have in https://github.com/rokucommunity/brighterscript/commit/bac0ae90bcc00df2689371e451983c33478c4695 ?

Reply:
no reason! I'll change it back